lib/libutee/arch/arm/user_ta_entry.c

```diff
@@ -187,10 +188,16 @@ static TEE_Result entry_invoke_command(unsigned long session_id,
 
 	__utee_to_param(params, &param_types, up);
 	ta_header_save_params(param_types, params);
-
+#ifdef CFG_TEE_BENCHMARK
+	tee_add_timestamp(params[TEE_BENCH_DEF_PARAM].memref.buffer,
```

Comment:
I also doubt about here, how can the fifth params pass to here, I can't see the fifth params type, and I look around the code, can't see the copy operation deal with this param, do I miss something?